dix/req.c

```diff
@@ -1418,11 +1436,12 @@ static void dix_item_rc_update(struct m0_dix_req  *req,
 		case DIX_GET:
 			m0_cas_get_rep(creq, key_idx, &get_rep);
 			rc = get_rep.cge_rc;
-			if (rc == 0) {
-				ditem->dxi_val = get_rep.cge_val;
-				/* Value will be freed at m0_dix_req_fini(). */
-				m0_cas_rep_mlock(creq, key_idx);
-			}
+			if (rc != 0) break;
+			if (!dix_item_version_cmp(ditem, get_rep)) break;
```

Comment:
Just a stylish proposal: may be it's better readable to use a single if statement without multiple breaks:
```
if (rc == 0 && dix_item_version_cmp(ditem, get_rep)) {
        .....
}
break;
```

Reply:
Changed